packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -1326,6 +1480,51 @@ curl -v -X POST \
 }
 ```
 
+#### Upload YubiKey OTP Seed
+
```

Comment:
Can we add this as well between the header and description?

`<ApiOperation method="post" url="/api/v1/org/factors/yubikey_token/tokens" />`